tests/model/test_session.py

```diff
@@ -42,3 +44,27 @@ def test_options(dismod):
     ))
     # Run a predict in order to verify the options are accepted.
     session.predict(model_var, avgints, parent_location)
+
```

Comment:
I like the bash test and /bin/false. I don't know what will happen when the run command fails b/c it isn't found. There's no error-condition testing. This is how it was, so I'm just commenting.

Reply:
It raises a FileNotFound exception. We could wrap it in a custom exception but it's really not a recoverable error the way this tool gets used so I think it's a fine choice to just let it bubble out.